From 9b3bc7ff3e701212842ed6ae582bbdd249877a18 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Wed, 4 Oct 2023 17:28:20 +0200
Subject: [PATCH 0569/1141] drm: bridge: adv7511: fill i2s stream capabilities

Set no_i2s_capture flag in hdmi_codec_pdata structure to report
that the ADV7535 HDMI bridge does not support i2s audio capture.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: Ie56518951348e20fdc6f945ec5943761a84e134b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/331691
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 drivers/gpu/drm/bridge/adv7511/adv7511_audio.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/gpu/drm/bridge/adv7511/adv7511_audio.c
+++ b/drivers/gpu/drm/bridge/adv7511/adv7511_audio.c
@@ -234,6 +234,7 @@ static const struct hdmi_codec_pdata cod
 	.ops = &adv7511_codec_ops,
 	.max_i2s_channels = 2,
 	.i2s = 1,
+	.no_i2s_capture = 1,
 	.spdif = 1,
 };
 
